3rdparty/libbacktrace/CMakeLists.txt

```diff
@@ -151,8 +151,13 @@ check_function_exists (strnlen HAVE_DECL_STRNLEN)
 
 check_function_exists (getexecname HAVE_GETEXECNAME)
 
+find_library(LIBUNWIND_LIBRARY NAMES unwind libunwind HINTS ${LIBUNWIND_ROOT}/lib ${LIBUNWIND_ROOT}/lib64 NO_DEFAULT_PATH)
 find_library(LIBUNWIND_LIBRARY NAMES unwind libunwind REQUIRE)
```

Comment:
isn't this line duplicating the line above? I.e. with your change in, isn't this line obsoleted? Simply add the REQUIRE above and then it should be fine?